base/data/gm4/tags/blocks/no_collision.json

```diff
@@ -0,0 +1,73 @@
+{
+  "values": [
+    "#gm4:air",
+    "#gm4:water",
+    "#minecraft:banners",
+    "#minecraft:buttons",
+    "#minecraft:corals",
+    "#minecraft:crops",
+    "#minecraft:flowers",
+    "#minecraft:portals",
+    "#minecraft:pressure_plates",
+    "#minecraft:rails",
+    "#minecraft:saplings",
+    "#minecraft:signs",
+    "#minecraft:wall_corals",
+    "minecraft:attached_melon_stem",
+    "minecraft:attached_pumpkin_stem",
+    "minecraft:beetroots",
+    "minecraft:brown_mushroom",
+    "minecraft:carrots",
+    "minecraft:cobweb",
+    "minecraft:crimson_fungus",
+    "minecraft:crimson_roots",
+    "minecraft:dead_brain_coral_fan",
+    "minecraft:dead_brain_coral_wall_fan",
+    "minecraft:dead_brain_coral",
+    "minecraft:dead_bubble_coral_fan",
+    "minecraft:dead_bubble_coral_wall_fan",
+    "minecraft:dead_bubble_coral",
+    "minecraft:dead_bush",
+    "minecraft:dead_fire_coral_fan",
+    "minecraft:dead_fire_coral_wall_fan",
+    "minecraft:dead_fire_coral",
+    "minecraft:dead_horn_coral_fan",
+    "minecraft:dead_horn_coral_wall_fan",
+    "minecraft:dead_horn_coral",
+    "minecraft:dead_tube_coral_fan",
+    "minecraft:dead_tube_coral_wall_fan",
+    "minecraft:dead_tube_coral",
+    "minecraft:fern",
+    "minecraft:fire",
```

Comment:
`minecraft:fire` and `minecraft:soul_fire` should be replaced with `#minecraft:fire`